x-pack/plugins/fleet/server/services/actions/types.ts

```diff
@@ -104,9 +104,9 @@ export interface FleetActionRequest extends CommonFleetActionResultDocFields {
     signature: string;
   };
   start_time?: string;
-  timeout: number;
+  timeout?: number;
   type: string;
-  user_id: string;
+  user_id: string | undefined;
```

Comment:
nit
```suggestion
  user_id?: string;
```

Reply:
Thanks for the suggestion @juliaElastic . This type is deliberate as I do want us to explicitly state when the `user_id` is `undefined`.

With `user_id?: string` the argument is optional (you may or may not assign `user_id` when creating a `FleetActionRequest` object) but with `user_id: undefined | T` one needs to explicitly state `undefined` as a key value in the `FleetActionRequest` object.